### PR TITLE
Add IBM i to package manager page

### PR DESCRIPTION
**Pull request by ThePrez**
Opening as a draft, as I figure it makes sense to "land" this when Node 12 is available from IBM (but am willing to take feedback in the meantime).

### CODE REVIEW

**Comment by Maledong** [comment]
@nodejs/website：Please have a review, everything is fine with me.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by srl295 — APPROVED** [review]

**Comment by aymen94** [comment]
LGTM

**Review by aymen94 — APPROVED** [review]

**Comment by ghost** [comment]
@nodejs/website：Please have a translation if possible for this, thanks anyway!

**Comment by yous** [comment]
It seems that `*` in "\*ALLOBJ" is not for italic: https://www.ibm.com/support/knowledgecenter/en/ssw_ibm_i_73/rzarl/rzarlallobjsa.htm

But `*` can be parsed as a part of markdown syntax, we should use `\*ALLOBJ special authority` instead of `*ALLOBJ special authority`.

**Comment by yous** [comment]
@nodejs/nodejs-ko Updated Korean translation. Please take a look.

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Comment by Aissaoui-Ahmed** [comment]
Update `package-manager.md` 
@celyes take a look !

File: locale/ar/download/package-manager.md [hunk 1]
```diff
@@ -73,6 +74,17 @@ cd /usr/ports/www/node && make install
 emerge nodejs
 ```
 
+## IBM i
+
+نسخ LTS لـNode.js متوفرة من IBM و متوفرة عبر [مدير الحزمة الـ'yum'](http://ibm.biz/ibmi-rpms). إسم الحزمة هو `nodejs` متبوعا برقم الإصدار الرائد (مثلا، `nodejs8`، `nodejs10`، `nodejs12`، إلخ
+
+لتثبيت Node.js 12.x من سطر الأوامر شغل التالي كمستخدم مع \*ALLOBJ  سلطة خاصة :
```
Comment (celyes) [inline locale/ar/download/package-manager.md]: change the line 81 to the following:

لتثبيت Node.js 12.x باستخدام سطر الأوامر، شغل الامر التالي كمستخدم مع سلطة \*ALLOBJ الخاصة :


Thank you so much ! :+1:

File: locale/ar/download/package-manager.md [hunk 2]
```diff
@@ -73,6 +74,17 @@ cd /usr/ports/www/node && make install
...
+
+ ```bash
+yum install nodejs12
+```
+يمكن أيضًا تثبيت Node.js مع IBM i وصول العملاء لحلول المنتج. انظر [هذه الوثيقة الدعم](http://www-01.ibm.com/support/docview.wss?uid=nas8N1022619) .لمزيد من التفاصيل
```
Comment (celyes) [inline locale/ar/download/package-manager.md]: Change the line 86 to the following : 

يمكن أيضًا تثبيت Node.js مع منتج IBM i الخاص بحلول وصول العملاء. انظر [وثيقة الدعم هذه](http://www-01.ibm.com/support/docview.wss?uid=nas8N1022619) لتفاصيل أكثر

thank you so much ^^ :+1:

**Review by celyes — CHANGES_REQUESTED** [review]

**Review by celyes — APPROVED** [review]
Looks great
thank you all @ThePrez @Aissaoui-Ahmed @Maledong @aymen94 @ouadie-lahdioui @mhdawson

**Comment by ghost** [comment]
Thank you all!

**Pull request merged**